### ISSUE TITLE
Support \sound

### ISSUE THREAD

**Issue by jspitz** [pdfpc/pdfpc#712]
Currently, `pdfpc` supports (a subset) of `multimedia`'s `\movie`, i.e. movie annotations, which also allows to play sound files (by design or by chance?). However, it does not yet seem to support sound objects (as generated by `multimedia`'s `\sound`. Since these have some advantages over movie annotations (see `beamer` docs), it would be nice if `pdfpc` would support this as well.

**Comment by fnevgeny** [comment]
What, specifically, do you find missing?

**Comment by jspitz** [comment]
The main differences, according to beamer docs, are:
1. When a page is closed, any playing movie is immediately stopped. Thus, you cannot use the `\movie` command to create sounds that persist for a longer time.
2. You cannot play two movies at the same time.

Moreover, I just find it appropriate to support this beamer command, as pdfpc is otherwise supporting beamer so well.

So if it is not too much of a hassle to implement support for this, I'd very much welcome it.

**Comment by jspitz** [comment]
Fair enough, it's not very urgent to me. I work around it by defining two separate output chains depending on whether I prepare a presentation to be presented with `pdfpc` or with a different PDF viewer.

**Comment by jspitz** [comment]
Yes, I do (sometimes). I also find it slightly more convenient if I play several (linguistic) sound examples in sequence that I do not have to wait until one is finished.

**Comment by fnevgeny** [comment]
> I also find it slightly more convenient if I play several (linguistic) sound examples in sequence that I do not have to wait until one is finished.

And are these from the same slide or different ones?

**Comment by jspitz** [comment]
Same slide. Typically phonetic minimal pairs (such as [cat]/[mat]) which are contrasted to each other.
